server/claim-review/dto/create-claim-review.dto.ts

```diff
@@ -9,15 +9,15 @@ import { Personality } from "../../personality/schemas/personality.schema";
 import { Claim } from "../../claim/schemas/claim.schema";
 
 export enum ClassificationEnum {
-    "not-fact",
-    "true",
-    "true-but",
-    "arguable",
-    "misleading",
+    "not-fact" = 1,
```

Comment:
I think we should leave `not-fact` as 0 and leave it out of the rating for ClaimReview schema, since not-fact is not important and is neither better/worst than the others.